Cargo.toml

```diff
@@ -90,6 +90,8 @@ serdect = "0.2.0"
 bls12_381 = "0.8.0"
 group = "0.13.0"
 itertools = "0.12.1"
+jsonrpsee-core = "0.22.5"
```

Comment:
![image](https://github.com/fedimint/fedimint/assets/133234413/100d4d5d-2c86-4c94-9975-befc86575e6c)
Do any of these other Cargo.toml's need to be updated?

Reply:
I only see the fedimint-api-client one changing.

Reply:
Will need to `workspace = true` in more places. Not touching it RN in this PR. :D

Reply:
https://github.com/fedimint/fedimint/issues/5248